README.md

```diff
@@ -1,7 +1,85 @@
 The goal of *mingwpy* project is to provide a free toolchain for building Python extensions for Windows OS, and make sure that those extensions are compatible with CPython compiled with native Visual Studio.
 
-*mingwpy* toolchain contains customized GCC compiler of the [GNU toolchain](https://en.wikipedia.org/wiki/GNU_toolchain) based on the [Mingw-w64](http://mingw-w64.org) and the [mingw-builds](https://github.com/niXman/mingw-builds) projects.
+*mingwpy* toolchain contains customized gcc, g++ and gfortran compilers of the [GNU toolchain](https://en.wikipedia.org/wiki/GNU_toolchain) based on the [Mingw-w64](http://mingw-w64.org) and the [mingw-builds](https://github.com/niXman/mingw-builds) projects.
 
  * Documentation: https://mingwpy.github.io/
  * Discussions: https://groups.google.com/forum/#!forum/mingwpy
- * Development: https://github.com/mingwpy/mingwpy/blob/master/mingwpy_builds_readme.md
+
+### Installing built mingwpy toolchain with pip
+
+Experimental builds of mingwpy for Python 2.7 and 3.4 are available for testing purposes:
+
+    pip install -i https://pypi.anaconda.org/carlkl/simple mingwpy
+
+Note: [Python 3.5 is not supported yet](https://mingwpy.github.io/issues.html#choice-of-msvc-runtime).
+
+### Using new toolchain with CPython
+
+To use the toolchain for CPython, see
+[readme_specs.md](https://github.com/mingwpy/mingwpy/blob/master/specs/readme_specs.md).
```

Comment:
Because those specs files are only useful in the context of build compiled extensions for **Python** via mingw**py** . The mingw-build project is a set of scripts that are generic (not python related at all) and it's better to keep it that way so that the maintenance effort can be kept as low as possible. We (mingwpy) don't want to maintain those scripts and want to let upstream mingw developers maintain those.

It's better to centralize the python specific stuff all in one place in the mingwpy repo that will be in charge of turning a generic mingw-build generated toolchain (either for 32 bit or 64 bit) into a toolchain that work with one specific version of Python.